util/travis/common.sh

```diff
@@ -31,14 +31,27 @@ install_linux_deps() {
 # Mac OSX build only
 install_macosx_deps() {
 	brew update
-	brew install freetype gettext hiredis irrlicht jpeg leveldb libogg libvorbis luajit
+	brew install freetype gettext hiredis irrlicht install leveldb libogg libvorbis luajit
+	brew upgrade jpeg # need to do an "upgrade", or build might fail
 	#brew upgrade postgresql
 }
 
 # Relative to git-repository root:
 TRIGGER_COMPILE_PATHS="src/.*\.(c|cpp|h)|CMakeLists.txt|cmake/Modules/|util/travis/|util/buildbot/"
 
+if [[ "$TRAVIS_BRANCH" != "" ]]; then #if $TRAVIS_BRANCH variable is set, enable function below
+	if [[ "$TRAVIS_BRANCH" != "$(git name-rev --name-only HEAD)" ]]; then
+		# ^ check if branch is same as current branch
```

Comment:
don't document <b>what</b>, document <b>why</b>

if it turns out that this code isn't needed (it shouldn't be), then remove it